pages/api/_peopleHR.ts

```diff
@@ -164,9 +164,9 @@ export async function getAllJobPostings(): Promise<JobPost[] | null> {
     return convertJSONToJobPosts(jobPostingData);
 }
 
-export async function getNumberOfActiveRoles(): Promise<number | undefined> {
+export async function getNumberOfActiveRoles(): Promise<number> {
     const allJobPosts = await getAllJobPostings();
-    if (!allJobPosts) {
+    if (!allJobPosts || !allJobPosts.length) {
         return 0;
     }
     return allJobPosts?.length;
```

Comment:
Happy for this to be commited in another PR to get this one merged```suggestion
    return allJobPosts.length;
```